src/test/common/test_crc32c.cc

```diff
@@ -266,3 +266,83 @@ TEST(Crc32c, RangeNull) {
     ASSERT_EQ(crc, *check);
   }
 }
+
+double estimate_clock_resolution()
```

Comment:
@branch-predictor But hey, now you got a result with an error estimation, how cool is THAT?